apple/testing/apple_test_rules.bzl

```diff
@@ -191,7 +191,7 @@ def _collect_files(rule_attr, attr_name):
         return depset()
 
     attr_val_as_list = attr_val if type(attr_val) == type([]) else [attr_val]
```

Comment:
I think this would be better to be refactored as:
```
return depset(transitive = [f.files for f in attr_val_as_list])
```

and remove the `files = ...` line. that way we won't expand the depsets, and should be a bit faster. (I wrote this, I'm a little ashamed 😛 )

Reply:
done!